HLTrigger/JetMET/plugins/HLTScoutingPFProducer.cc

```diff
@@ -194,19 +237,15 @@ void HLTScoutingPFProducer::produce(edm::StreamID sid, edm::Event &iEvent, edm::
       }
       //get the PF constituents of the jet
       std::vector<int> candIndices;
-      if (doCandidates) {
-        for (auto &cand : jet.getPFConstituents()) {
-          if (not(cand.isNonnull() and cand.isAvailable())) {
-            throw cms::Exception("HLTScoutingPFProducer")
-                << "invalid reference to reco::PFCandidate from reco::PFJet::getPFConstituents()";
-          }
```

Comment:
I don't think this check should be removed (see https://github.com/cms-sw/cmssw/pull/35889).

Reply:
I added it back.